_includes/data-access-table.html

```diff
@@ -61,24 +61,27 @@
     <div class="arrow arrow-down"></div>
   </div> 
   <div>
-    Identify the Data Access Committee (DAC) affiliated with the dataset(s)
-    you wish to request access for. Then, contact the DAC to start your data
-    access application.
+    Identify the Data Access Committee (DAC) affiliated with the dataset(s) you
+    wish to request access for. Then, contact the DAC to start your data access
+    application. A Data Access Agreement (DAA) documenting the terms of access
+    and usage of the dataset needs to be negotiated between you and the DAC and
+    should be signed by both parts.
   </div>
 
   <div class="steps">3. Receive download instructions
     <div class="arrow arrow-down"></div>
   </div> 
   <div>
-    Once the Data Access Committe (DAC) has accepted your data access request,
-    you will recieve instructions from the SDA Helpdesk on how to access the
-    data.
+    Once the DAC has accepted your data access request and an authorisation
+    grant has been received by the SDA Helpdesk, you will recieve instructions
+    from the SDA Helpdesk on how to access the data.
   </div>
 
   <div class="steps">4. Download and decrypt files</div>
   <div>
-    Download and decrypt the files according to the instructions provided 
-    by the SDA Helpdesk.
+    Download and decrypt the files according to the instructions provided by
```

Comment:
Should we include something about decrypting in a secure environment?

Reply:
I have added that in the GitHub download and decrypt instructions.

Reply:
Yes, that would be good. @shreyasshivakumara has prepared a text, but I don't think he has added it yet to https://github.com/NBISweden/EGA-SE-user-docs

Reply:
Yes! It will be updated in 20 minutes.

Reply:
Link to the SDA-specific instructions has now been added